scripts/pack-release.js

```
@@ -46,7 +51,55 @@ async function pack() {
     }
   }
 
-  console.log('\n✨ All releases packed in /releases folder');
+  // ── 2. Pack VS Code extension (.vsix) ───────────────────
+  console.log('\n📦 Packing VS Code extension...');
+  const vscodePath = path.resolve(root, 'packages/vscode');
+
+  // Build the extension
+  console.log('🔨 Building VS Code extension...');
```

ESLint

[error] 'console' is not defined.

(no-undef)